examples/basic-globalize-compiler/README.md

```diff
@@ -0,0 +1,26 @@
+# Basic Globalize Compiler example
+
+On this example, we assume you know the Globalize usage basics. So, it focuses on
+the Globalize Compiler and on the Globalize runtime modules.
+
+## Running the example
+
+### Development mode
+
+1. Point your browser at `./development.html`. Note that the formatters are
+created dynamically. Therefore, Cldrjs and CLDR data are required.
+1. Understand the demo by reading the source code. We have comments there for
+you.
+
+### Production mode
+
+1. Install `globalize-compiler` by running `npm install`.
+1. Compile the application formatters by running `npm run build`. See
```

Comment:
`0.` maybe makes it more clear that we're using this trick. Though I think it's alright as-is.